### ISSUE TITLE
[Nethereum.Parity] Implement parity_listStorageKeys

### ISSUE THREAD

**Issue by Texnomic** [Nethereum/Nethereum#774]

**Comment by juanfranblanco** [comment]
@Texnomic Parity is getting deprecated (the client)

**Comment by Texnomic** [comment]
Correct, but the API Calls have been migrated to [Erigon](https://github.com/ledgerwatch/erigon ) using the same namespace

**Comment by juanfranblanco** [comment]
Ah thank you! I did not know your use case, I would use the Erigon client for testing now too, instead of OE.

**Comment by juanfranblanco** [comment]
I can see it now https://github.com/ledgerwatch/erigon/releases/tag/v2022.01.02

**Comment by Texnomic** [comment]
Actually you can use it now for mainnet. it is almost perfect. Archive Node in under 2 TB, it worth it's own Nethereum.Erigon NuGet honestly :)

**Comment by juanfranblanco** [comment]
Yes, it will require its own package as it is going to develop furthermore than parity and OE, whilst keeping the other for backwards compatibility. I need to find a good way to reuse the none standard RPC methods like "parity_" etc. Keep thinking about it.
